src/platforms/hosted/ftdi_bmp.h

```diff
@@ -33,12 +33,20 @@
 
 #include "probe_info.h"
 
-typedef struct data_desc {
-	uint16_t data_low;
-	uint16_t ddr_low;
-	uint16_t data_high;
-	uint16_t ddr_high;
-} data_desc_s;
+/*
+ * This structure defines a current or desired port state. Ports are 16-bit, though
+ * on the 4-port FTDI models the upper 16 bits are not bonded out.
```

Comment:
Maybe?

```suggestion
 * on the 4-port FTDI models the upper 16 bits are not bound.
```

Reply:
"bonded out" refers here to being physically wire bonded from the silicon die to the pins on the package. We did get the bit count wrong though - "upper 8 bits".